components/global/node-blog-teaser.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <nuxt-link class="node" :to="path.pathAlias">
+    <h2 v-if="title" class="mb-2 text-3xl font-semibold">{{ title }}</h2>
```

Comment:
I think the naked starter should stick with defaults as we have it in nuxtjs-drupal-ce

we need to think on how to support drupal-cms best. maybe create a separate starter? or ship components with our recipes and copy it over to the frontend when installing the recipe?